.travis.yml

```diff
@@ -25,11 +25,19 @@ matrix:
   - php: hhvm
 
 before_script:
+  - |
+    # Remove Xdebug for performance increase:
```

Comment:
Why?! Is the difference of test-time so huge? 
By this change, we lose any opportunity to test code coverage and errors deep analysis.